lib/views/index.erb

```diff
@@ -0,0 +1,18 @@
+<section>
+<a href="users/listings">My Listings</a>
+
+</section>
+<h1> Welcome to MakersBnb! </h1>
+
+<ul id='post'>
+
+	<% @spaces.each do |spaces| %>
+	<li>
+		Location: <%= spaces.location %> 
+		Name: <a href= '<%= spaces.id %>'><%= spaces.name %> </a>   
+		Description: <%= spaces.description %>
+		Price: <%= spaces.price %>
```

Comment:
Spaces, not tabs! Pick on this sort of stuff in code review, it shouldn't pass muster.